hw-model/src/lib.rs

```diff
@@ -250,6 +250,7 @@ pub struct BootParams<'a> {
     pub initial_repcnt_thresh_reg: Option<CptraItrngEntropyConfig1WriteVal>,
     pub initial_adaptp_thresh_reg: Option<CptraItrngEntropyConfig0WriteVal>,
```

Comment:
Did you forget to remove init_params from this struct?

Reply:
I did forget this... I did all the work to separate them then missed the last step. Thank you for catching it!

Reply:
I have pushed a new patch that removes this